pdns/dnsdistdist/docs/reference/config.rst

```diff
@@ -1651,6 +1651,14 @@ Other functions
 
   :param int size: The maximum size in bytes (default is 512)
 
+.. function:: setTCPFastOpenKey(key)
+
+  .. versionadded:: 1.7.1
+
+  Set a specified same ``TCP Fast Open`` key for all frontends.
```

Comment:
```suggestion
  .. versionadded:: 1.8.0

  Set the supplied ``TCP Fast Open`` key on all frontends. This can for example be used to allow all dnsdist instances in an anycast cluster to use the same ``TCP Fast Open`` key, reducing round-trips.
```